docker-compose.yml

```diff
@@ -90,23 +92,23 @@ x-snuba-defaults: &snuba_defaults
     SNUBA_STATSD_PORT: # Example value: 8125
 services:
   smtp:
-    <<: *restart_policy
+    <<: [*restart_policy, *pull_policy]
```

Comment:
I don't think we want this pull policy on images that we don't build, right @aldy505?

Reply:
Yes, I would remove the `*pull_policy` for this container.